playbooks/roles/appsembler_reporting/tasks/main.yml

```diff
@@ -0,0 +1,27 @@
+---
+
+- name: make logs directory for appsembler_reporting
```

Comment:
@bryanlandia I don't think there's a need for `.sh` files here. We can directly create cronjobs as have been done in:

 - https://github.com/appsembler/edx-configs/blob/master/easycare/prod/files/customer-post.yml

Reply:
@OmarIthawi Arrgh, I hate repeating work that's already been done :(  Yes, agreed about the shell scripts—I was just decided to replicate what @johnbaldwin had set up on ASU and keep them.  Your playbook is cleaner, but as this is something which will be replicated for multiple customers I like having a role.  What do you think?

Reply:
@bryanlandia work duplication is inevitable. The other, worse alternative is spamming everyone about everything you do. So please don't worry about it.

> as this is something which will be replicated for multiple customers I like having a role. What do you think?

You're right, having a playbook is much better than copying the file to every customer. So, please copy as much you'd like from `easycare/prod/files/customer-post.yml` and make it a proper role that we can reuse for others.

`customer-post.yml` should be used for customer-specific changes and hacks.

Reply:
@OmarIthawi Great.  Yes, let's use `customer-post.yml` like for Trinity's case where there is (I think) an additional report to run, but everything else in the role.  I'll update the role here.